wrappers/services.go

```diff
@@ -1570,22 +1570,40 @@ type RestartServicesFlags struct {
 }
 
 // Restart or reload services; if reload flag is set then "systemctl reload-or-restart" is attempted.
```

Comment:
this doc-comment needs to explain the explicitServices argument now

Reply:
Done